chainer/datasets/kmnist.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+import numpy
+
+import chainer
+from chainer.dataset import download
+from chainer.datasets._mnist_helper import make_npz
+from chainer.datasets._mnist_helper import preprocess_mnist
+
+
+_kmnist_labels = [('o', u'\u304A'), ('ki', u'\u304D'),
+                  ('su', u'\u3059'), ('tsu', u'\u3064'),
+                  ('na', u'\u306A'), ('ha', u'\u306F'),
+                  ('ma', u'\u307E'), ('ya', u'\u3084'),
+                  ('re', u'\u308C'), ('wo', u'\u3092')]
+
+
+def get_kmnist_labels():
+    """Provide a list of labels for the KMNIST dataset.
+
+    Returns:
+        List of labels in the form of tuples. Each tuple contains the
+        character name in romaji as a string value and the unicode codepoint
+        for the character.
+
+    """
+    return list(_kmnist_labels)
```

Comment:
I think it's okay to skip this list casting, since it's already a list.

Reply:
> I'm wondering if we should use the more verbose "kuzushiji_mnist" instead of "kmnist" if we we're to provide "kuzushiji_49" and "kuzushiji_kanji" in the future, for consistency that is. Another argument for this would be that "fashion_mnist" already follows this naming convention. @wintercarver what do you think?

I am OK with this. I actually started with "kuzushiji_" on everything but it felt a little long, however it would be more clear and "k" is not a very informative prefix. If you agree, I'll make the changes of `kmnist` -> `kuzushiji_mnist`.

Reply:
I do agree!